examples_tests/49.Radix_Sort/compute_shader.comp

```diff
@@ -4,14 +4,15 @@
 #version 430 core
 #define G_IDX gl_GlobalInvocationID.x
 #define G_IDY gl_GlobalInvocationID.y
-#define LC_IDX gl_LocalInvocationIndex
+#define LC_IDX gl_LocalInvocationIndex.x
 
 
-layout(local_size_x = 10) in;
+layout(local_size_x = 10, local_size_y = 1, local_size_z = 1) in;
```

Comment:
you want your whole workgroup size to be a multiple of 64, usually 256 is ideal

Reply:
I know, but It is currently for test purposes (understanding what I am doing). Finally will be divisible by 64